### DISCUSSION TITLE
[761830]  末日准备狂 Mr. Prepper（本体）

### DISCUSSION THREAD

**Discussion by Dew778** [pjy612/SteamManifestCache#380] [Share]
`
appid：761830
`

```
761831;46eeb5fdced56fe223ab7a3ee5efbf1cfc9f81ab82d1285f0fe38b1ccb2daf6c
761832;77abf0e1403cf75ac402c4a621736b0a475028a83ac1b8b94dcac5b2d97d8232
```

[761830_config.vdf.txt](https://github.com/pjy612/SteamManifestCache/files/13973094/761830_config.vdf.txt)
[[761830] 末日准备狂 Mr. Prepper（本体）清单.zip](https://github.com/pjy612/SteamManifestCache/files/13973123/761830.Mr.Prepper.zip)

Code：7337BCEA697F2EBE
Email:3394748293@qq.com

**Comment by ZEERDEER** [comment]
标题格式 #26

**Reply by Dew778** [reply]
已修改

**Reply by pjy612** [reply]
查收github邮件
